This is causing bunch of test failures
```
[INFO] 
[INFO] Results:
[INFO] 
[ERROR] Failures: 
[ERROR]   CatalogTest>AbstractTestNGSpringContextTests.run:154->testCatalog:140
[ERROR]   FutureCreditOfferTest>AbstractTestNGSpringContextTests.run:154->testFutureCreditAllOfferTypes:196 » IllegalArgument attempt to create merge event with null entity
[ERROR]   FutureCreditOfferTest>AbstractTestNGSpringContextTests.run:154->testFutureCreditAndOrderDiscountOrderAndOrderItemOffers:164 » IllegalArgument attempt to create merge event with null entity
[ERROR]   FutureCreditOfferTest>AbstractTestNGSpringContextTests.run:154->testFutureCreditAndOrderDiscountPercentOffOffers:136 » IllegalArgument attempt to create merge event with null entity
[ERROR]   FutureCreditOfferTest>AbstractTestNGSpringContextTests.run:154->testTwoFutureCreditPercentOffOffers:108 » IllegalArgument attempt to create merge event with null entity
[ERROR]   OfferAuditTest>AbstractTestNGSpringContextTests.run:154->testMinimumDaysPerUsageAudit:110 » IllegalArgument attempt to create merge event with null entity
[ERROR]   OfferTest>AbstractTestNGSpringContextTests.run:154->testBOGOAmountOffCombination:188 » IllegalArgument attempt to create merge event with null entity
[ERROR]   OfferTest>AbstractTestNGSpringContextTests.run:154->testBOGOCombination:169 » IllegalArgument attempt to create merge event with null entity
[ERROR]   OfferTest>AbstractTestNGSpringContextTests.run:154->testOfferUsedForPricing:263 » IllegalArgument attempt to create merge event with null entity
[ERROR]   OfferTest>AbstractTestNGSpringContextTests.run:154->testPercentOffOfferWithItemMinPriceQuantityReduction:242 » IllegalArgument attempt to create merge event with null entity
[ERROR]   OfferTest>AbstractTestNGSpringContextTests.run:154->testPercentOffOfferWithItemMinPriceSecondEvaluation:224 » IllegalArgument attempt to create merge event with null entity
[ERROR]   OfferTest>AbstractTestNGSpringContextTests.run:154->testPercentOffOfferWithScaleGreaterThanTwo:209 » IllegalArgument attempt to create merge event with null entity
[ERROR]   OfferTest>AbstractTestNGSpringContextTests.run:154->testThreePercentOffOffers:130 » IllegalArgument attempt to create merge event with null entity
[ERROR]   OfferTest>AbstractTestNGSpringContextTests.run:154->testTwoPercentOffOffers:150 » IllegalArgument attempt to create merge event with null entity
[INFO] 
[ERROR] Tests run: 160, Failures: 14, Errors: 0, Skipped: 25
```

--- a/pom.xml
+++ b/pom.xml
@@ -57,7 +57,7 @@
         <groovy.version>4.0.20</groovy.version>
         <gpg.plugin.version>3.1.0</gpg.plugin.version>
         <broadleaf-presentation.version>2.0.0-GA</broadleaf-presentation.version>
-        <hsqldb.database.starter.version>3.0.0-GA</hsqldb.database.starter.version>
+        <hsqldb.database.starter.version>3.0.1-SNAPSHOT</hsqldb.database.starter.version>
         <hsqldb.version>2.7.2</hsqldb.version>
         <closure.compiler.version>v20180506</closure.compiler.version>
         <hibernate.version>6.2.13.Final</hibernate.version>

